stories/Molecules/Form/signup.scss

```diff
@@ -40,12 +40,13 @@
 
   .form-submit {
     cursor: pointer;
-    font-family: $Font-Family-ProximaBold;
+    font-family: $Font-Family-ProximaNova;
```

Comment:
@harishkumar-srijan do we need to mention font family here

Reply:
Done